### DISCUSSION TITLE
intermediate /data.json  file is empty

### DISCUSSION THREAD

**Discussion by damarudhvarma** [Kushal997-das/Hacktoberfest-2021#248] [Q&A]
what details should i fill and what is the format to that

**Comment by neyhere07** [comment]
> what details should i fill and what is the format to that

Hello @damarudhvarma! Since this repository has been excluded from Hacktoberfest 2024, a new repository named "[Skill Show](https://github.com/Kushal997-das/SkillShow)" has been created where you can contribute. Read all the details in the contributing guidelines carefully.
